docs/manuals/gui/viselements/charts/filled-area-plots.md

```diff
@@ -0,0 +1,155 @@
+## Filled Area Plots
+
+An area chart displays a solid color between the traces of a graph. It is an option on scatter charts. Areas will be filled according to the ‘fill’ option ("none", "tozeroy", "tozerox" , "tonexty", "tonextx", "toself", and "tonext")
```

Comment:
=>
An area chart displays a solid colored area between the traces of a graph.

You can create a filled plot by using a *scatter* chart type, and specifying the `fill` option of the chart *options* dictionary.
This value can be one of "none", "tozeroy", "tozerox" , "tonexty", "tonextx", "toself", and "tonext", as indicated in the [Plotly documentation](https://plotly.com/python/reference/#scatter-fill).